R/assertions.R

```diff
@@ -1665,6 +1665,8 @@ assert_one_to_one <- function(dataset, vars1, vars2) {
 #' @param var_name The name of the variable. If the argument is specified, the
 #'   specified name is displayed in the error message.
 #'
+#' @inheritParams assert_logical_scalar
```

Comment:
We can't take the documentation of `message` from `assert_logical_scalar()` because there is no `arg_name` argument.
![image](https://github.com/pharmaverse/admiraldev/assets/80953585/bb11cc43-a6dc-43ad-8efe-309157e0208d)

Reply:
good catch, thank you!. updated!